File: Calcular Frete/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using WindowsFormsApplication1.br.com.correios.ws;

namespace WindowsFormsApplication1
{
    public partial class Form1 : Form
    {
        Frete freteagr = new Frete();
        cResultado resultado;
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            freteagr.Comprimento =Convert.ToDecimal(textBox1.Text);
            freteagr.Largura = Convert.ToDecimal(textBox2.Text);
            freteagr.Altura = Convert.ToDecimal(textBox3.Text);
            freteagr.Peso = textBox4.Text;
            freteagr.CEP = textBox5.Text;
            resultado = freteagr.calcular();
            MessageBox.Show("Sedex Varejo \n Preço:" + resultado.Servicos.ElementAt(0).Valor + "\n Prazo em dias:" + resultado.Servicos.ElementAt(0).PrazoEntrega + "\n" + resultado.Servicos.ElementAt(0).MsgErro + "\n Sedex A Cobrar Varejo \n Preço:" + resultado.Servicos.ElementAt(1).Valor + "\n Prazo em dias:" + resultado.Servicos.ElementAt(1).PrazoEntrega + "\n" + resultado.Servicos.ElementAt(1).MsgErro + "\n Sedex 10 Varejo \n Preço:" + resultado.Servicos.ElementAt(2).Valor + "\n Prazo em dias:" + resultado.Servicos.ElementAt(2).PrazoEntrega + "\n" + resultado.Servicos.ElementAt(2).MsgErro + "\nSedex Hoje Varejo \n Preço:" + resultado.Servicos.ElementAt(3).Valor + "\n Prazo em dias:" + resultado.Servicos.ElementAt(3).PrazoEntrega + "\n" + resultado.Servicos.ElementAt(3).MsgErro + "\n PAC Varejo \n Preço:" + resultado.Servicos.ElementAt(4).Valor + "\n Prazo em dias:" + resultado.Servicos.ElementAt(4).PrazoEntrega + "\n" + resultado.Servicos.ElementAt(4).MsgErro);
        }
    }
}

File: Calcular Frete VB/Form1.vb
Public Class Form1
    Public Frete As frete = New frete


    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        If TextBox1.Text <> "" Or
           TextBox2.Text <> "" Or
           TextBox3.Text <> "" Or
           TextBox4.Text <> "" Or
           TextBox5.Text <> "" Then
            Frete.Largura = System.Convert.ToDecimal(TextBox1.Text)
            Frete.Comprimento = System.Convert.ToDecimal(TextBox2.Text)
            Frete.Altura = System.Convert.ToDecimal(TextBox3.Text)
            Frete.Peso = TextBox4.Text
            Frete.CEP = TextBox5.Text
            Frete.calcular()
            MessageBox.Show("SEDEX Varejo" + Chr(13) + "Preço: " + Frete.Resultado.Servicos.ElementAt(0).Valor + Chr(13) + "Prazo em dias: " + Frete.Resultado.Servicos.ElementAt(0).PrazoEntrega + Chr(13) + Frete.Resultado.Servicos.ElementAt(0).MsgErro + Chr(13) + "SEDEX A Cobrar Varejo" + Chr(13) + "Preço: " + Frete.Resultado.Servicos.ElementAt(1).Valor + Chr(13) + "Prazo em dias: " + Frete.Resultado.Servicos.ElementAt(1).PrazoEntrega + Chr(13) + Frete.Resultado.Servicos.ElementAt(1).MsgErro + Chr(13) + "SEDEX 10 Varejo" + Chr(13) + "Preço: " + Frete.Resultado.Servicos.ElementAt(2).Valor + Chr(13) + "Prazo em dias: " + Frete.Resultado.Servicos.ElementAt(2).PrazoEntrega + Chr(13) + Frete.Resultado.Servicos.ElementAt(2).MsgErro + Chr(13) + "SEDEX Hoje Varejo" + Chr(13) + "Preço: " + Frete.Resultado.Servicos.ElementAt(3).Valor + Chr(13) + "Prazo em dias: " + Frete.Resultado.Servicos.ElementAt(3).PrazoEntrega + Chr(13) + Frete.Resultado.Servicos.ElementAt(3).MsgErro + Chr(13) + "PAC Varejo" + Chr(13) + "Preço: " + Frete.Resultado.Servicos.ElementAt(4).Valor + Chr(13) + "Prazo em dias: " + Frete.Resultado.Servicos.ElementAt(4).PrazoEntrega + Chr(13) + Frete.Resultado.Servicos.ElementAt(4).MsgErro)



        End If
    End Sub
End Class
